custom-resource-handlers/src/_rmrf.ts

```diff
@@ -0,0 +1,15 @@
+import fs = require('fs');
+import path = require('path');
+import util = require('util');
+
+export = async function _rmrf(filePath: string): Promise<void> {
```

Comment:
fs-extra?

Reply:
If I take dependencies, I have to be certain they don't (and will never) include native code (otherwise I need to make sure it'd built for the Lambda container OS/architecture).

Reply:
Makes sense